plugins/flutter_plugins/url_launcher_fde/linux/url_launcher_plugin.cc

```diff
@@ -44,6 +44,11 @@ class UrlLauncherPlugin : public flutter::Plugin {
       std::unique_ptr<flutter::MethodResult<EncodableValue>> result);
 };
 
+// utility function to compare url schemes
```

Comment:
How about "Returns true if |s| starts with |prefix|.". (The current comment says what it's used for, not what it does.)